src/CopyrighFixer/CopyrighFixer/copyrighfixer.h

```diff
@@ -14,4 +15,25 @@ namespace CopyrighFixer {
 
 bool CopyrighFixer_EXPORT init();
 
+/**
+ * @brief The InfoOwner class for collect information about owner.
+ */
+class CopyrighFixer_EXPORT InfoOwner{
+
+public:
+    InfoOwner();
+
+    /**
+     * @brief name - this is name of the owner.
+     */
+    QString name;
+
+
+    /**
+     * @brief timeRange - this is a string value indicating the  dates of ownership of the file.
+     */
+    QString timeRange;
```

Comment:
1. You must make the name and timeRange fields to private. Because you make **InfoOwner** as a class.
2. Create a getters and setters methods for all fields of this class. 
3. You must be documents only public members of a class i.e. getters and setters.